tapir/shifts/templates/shifts/user_shifts_overview_tag.html

```diff
@@ -129,18 +129,33 @@ <h5 class="card-header d-flex justify-content-between align-items-center" id="us
-            <div class="col-8 d-flex gap-1">
-                <form method="post" action="{% url 'shifts:solidarity_shift_used' object.pk%}">
-                    {% csrf_token %}
-                    <button type="submit" class="{% tapir_button_action %} btn-sm {% disabled_if_user_cant_receive_solidarity user %}"><span class="material-icons">favorite</span>{% translate 'Receive Solidarity' %}</button>
-                </form>
-                <form method="post" action="{% url 'shifts:solidarity_shift_given' object.pk%}">
-                    {% csrf_token %}
-                    <button type="submit" class="{% tapir_button_action %} btn-sm {% disabled_if_user_cant_give_solidarity user %}"><span class="material-icons">favorite</span>{% translate 'Give Solidarity' %}</button>
-                </form>
+        {% feature_flag_enabled "feature_flag_solidarity_shifts" as solidarity_shifts_enabled %}
+        {% if solidarity_shifts_enabled %}
+            <div class="row m-1">
+                <div class="col-4 fw-bold text-end">{% translate "Solidarity" %}:</div>
+                <div class="col-8 d-flex gap-1">
+                    <form method="post" action="{% url 'shifts:solidarity_shift_used' object.pk%}">
+                        {% csrf_token %}
+                        <button type="submit" class="{% tapir_button_action %} btn-sm {% disabled_if_user_cant_receive_solidarity user %}"><span class="material-icons">favorite</span>{% translate 'Receive Solidarity' %}</button>
+                    </form>
+                    <form method="post" action="{% url 'shifts:solidarity_shift_given' object.pk%}">
+                        {% csrf_token %}
+                        <button type="submit" class="{% tapir_button_action %} btn-sm {% disabled_if_user_cant_give_solidarity user %}"><span class="material-icons">favorite</span>{% translate 'Give Solidarity' %}</button>
+                    </form>
+                </div>
             </div>
-        </div>
+            <div class="row m-1">
+                <div class="col-4 fw-bold text-end">{% translate "Solidarity Status" %}:</div>
+                <div class="col-8 d-flex">
+                    {% if user.shift_user_data.get_used_solidarity_shifts_current_year >= 2 %}
+                        <p>You already used {{ user.shift_user_data.get_used_solidarity_shifts_current_year }} out of 2 Solidarity Shifts this year</p>
+                    {% elif user.shift_user_data.get_used_solidarity_shifts_current_year < 2 and user.shift_user_data.get_available_solidarity_shifts and user.shift_user_data.get_account_balance < 0 %}
+                        <p>There are Solidarity Shifts available for you to use. You used {{ user.shift_user_data.get_used_solidarity_shifts_current_year }} out of 2 Solidarity Shifts this year</p>
+                    {% elif not user.shift_user_data.get_available_solidarity_shifts %}
+                        <p>There are no Solidarity Shifts available at the moment</p>
```

Comment:
We need to wrap each paragraph in `{% blocktranslate with ... %}` for translations. 
Using `with inner_variable=outer_variable` makes it less error prone for other users when they edit the translation file.
Example here : https://github.com/SuperCoopBerlin/tapir/blob/87038bf5a804bc6fbd8c3440344342426ec7e633/tapir/accounts/templates/accounts/edit_username.default.html#L16-L18